src/hb-coretext.cc

```diff
@@ -538,6 +545,18 @@ _hb_coretext_shape (hb_shape_plan_t    *shape_plan,
 	      CFNumberCreate (kCFAllocatorDefault, kCFNumberIntType, &active_features[j].rec.feature),
 	      CFNumberCreate (kCFAllocatorDefault, kCFNumberIntType, &active_features[j].rec.setting)
 	    };
+#else
+	    char tag[4];
+	    hb_tag_to_string (active_features[j].rec.feature, tag);
```

Comment:
Need to make room for a nul byte in tag and setting to zero no?  `hb_tag_to_string` just writes four bytes.

Reply:
You can also do `char tag[5] = {HB_UNTAG (active_features[j].rec.feature)};`.

Reply:
Much simpler. Done.